### ISSUE TITLE
Dropdown values 

### ISSUE THREAD

**Issue by fridzema** [Semantic-Org/Semantic-UI#6266]
Hello, 

I am facing a strange problem. I have a simple dropdown on a page where i set the values in html (if there is any).
There was a specific page where i didn't look for a while and worked perfectly in the past. I have a dropdown on that page with the following configuration:
```js
    $(filter).dropdown({
      apiSettings: {
        url: '/api/select/' + relation + '/{query}'
      }
    })
```
As you can see most minimal setup for remote content. 

In version 2.2.13 this is working fine and selected all the values defined in the html on page load. 
If i update to a higher version this behaviour is broken and nothing gets selected. I have tried with the saveRemoteData option, set selected etc.  

Any idea what the problem is here?

**Comment by fridzema** [comment]
This i my complete setup at the moment:
https://jsfiddle.net/ca0rovs3/309/

Can't get this example really to work because of api results. 
But the idea is that if select a result, you submit the form and you come back the values are remembered by semantic ui. The text of the values are appended to the html, this behaviour is what i am looking for and this works in 2.2.13 and lower only....

I hope it is a bit clear now?

**Comment by prudho** [comment]
This looks related with the famous #6123 bug. You'll have to wait for @jlukic new version with a fix, or build your own semantic-ui with this fix: d980e1dba622e1551329e7bcb155ce8e2ad7ae87

**Comment by fridzema** [comment]
Sorry, i've tried to find a similar question but couldn't find it. 
This looks like my problem indeed, thanks!
